index.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference path='typings/tsd.d.ts' />
+/// <reference path='typings/definitions.d.ts' />
```

Comment:
Ahh!  I get what you are saying now.

It does compile right now only because it is getting the type definitions from another import :).

I guess the question is whether we should scope overrides closest to usage or the third-party type definitions.  What do you think?